jvm-packages/pom.xml

```diff
@@ -52,7 +53,65 @@
         <module>xgboost4j-spark</module>
         <module>xgboost4j-flink</module>
     </modules>
+
     <profiles>
+        <profile>
+            <!-- default active profile excluding gpu related test suites -->
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.scalatest</groupId>
+                        <artifactId>scalatest-maven-plugin</artifactId>
+                        <configuration>
+                            <tagsToExclude>ml.dmlc.xgboost4j.java.GpuTestSuite</tagsToExclude>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <!-- gpu profile with both cpu and gpu test suites -->
+        <profile>
+            <id>gpu</id>
+            <activation>
+                <property>
+                    <name>use.cuda</name>
+                    <value>ON</value>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.scalatest</groupId>
+                        <artifactId>scalatest-maven-plugin</artifactId>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <!-- gpu-with-gpu-tests profile with only gpu test suites -->
+        <profile>
+            <id>gpu-with-gpu-tests</id>
```

Comment:
so we really need two profile? mvn does support something like testing a particular suite

Reply:
Hi @CodingCat, Thx for the review. 

I suppose "mvn does support something like testing a particular suite" is specifying like 
```
 mvn test -Dsuites=ml.dmlc.xgboost4j.scala.DMatrixSuite
```

Yeah, Seems mvn can specify which test suite to run, Suppose the test suite **must be** package name. So if PR has added another gpu related suites, then developer needs to change the CI build script to add another gpu suite.

This profile is only for CI used, and it will pick any suite tagged with GpuTestSuite to run. So the only thing for developer to do is just adding a GpuTestSuite tag to the new gpu suites, developer does not need to touch CI part,  then everything will work as expected.